clsCommon.pri

```diff
@@ -32,6 +39,16 @@ HEADERS *=\
 	source/dataman/info/CLSOceanOptics65000DetectorInfo.h \
 	source/dataman/info/CLSAmptekSDD123DetectorInfo.h \
 	source/dataman/info/CLSSynchronizedDwellTimeConfigurationInfo.h \
+	source/ui/CLS/CLSValueProgressLabel.h \
+	source/ui/CLS/CLSValueEditor.h \
+	source/ui/CLS/CLSControlEditor.h \
```

Comment:
@helfrij @iainworkman 

There is another issue #2127, which might handle this.